acceptance/acceptance_test.go

```diff
@@ -482,7 +482,9 @@ func testAcceptance(
 
 	it.Before(func() {
 		pack = invoke.NewPackInvoker(t, assert, subjectPackConfig, registryConfig.DockerConfigDir)
+		pack.EnableExperimental()
```

Comment:
At first I was going to suggest conditionalizing this on the daemon host OS being Windows, but I think this is the right call.